package.json

```diff
@@ -16,71 +16,69 @@
+    "@alaskaairux/auro-flightline": "^1.4.0",
     "chalk": "^4.1.2",
-    "lit-element": "^2.5.1",
-    "lit-html": "^1.4.1"
+    "lit": "^2.7.3"
   },
   "peerDependencies": {
     "@alaskaairux/auro-datetime": "^1.3.0",
     "@alaskaairux/design-tokens": "^3.9.1",
     "@alaskaairux/webcorestylesheets": "^3.7.3"
   },
   "devDependencies": {
-    "@alaskaairux/auro-datetime": "^1.3.0",
-    "@alaskaairux/design-tokens": "^3.11.0",
-    "@alaskaairux/icons": "^4.14.0",
+    "@alaskaairux/auro-datetime": "^1.4.0",
+    "@alaskaairux/design-tokens": "^3.13.0",
+    "@alaskaairux/icons": "^4.26.0",
     "@alaskaairux/webcorestylesheets": "^3.7.3",
     "@aurodesignsystem/eslint-config": "^1.3.0",
-    "@commitlint/cli": "^17.0.3",
-    "@commitlint/config-conventional": "^17.0.3",
-    "@open-wc/testing": "^3.1.6",
-    "@rollup/plugin-commonjs": "^22.0.1",
-    "@rollup/plugin-node-resolve": "^13.3.0",
-    "@semantic-release/changelog": "^6.0.1",
+    "@commitlint/cli": "^17.6.1",
+    "@commitlint/config-conventional": "^17.6.1",
+    "@open-wc/testing": "^3.1.8",
+    "@rollup/plugin-node-resolve": "^15.0.2",
+    "@semantic-release/changelog": "^6.0.3",
     "@semantic-release/git": "^10.0.1",
-    "@semantic-release/npm": "^9.0.1",
-    "@web/dev-server": "^0.1.32",
-    "@web/test-runner": "^0.13.31",
-    "autoprefixer": "^10.4.7",
-    "babel-loader": "^8.2.5",
+    "@semantic-release/npm": "^10.0.3",
+    "@types/estree": "^1.0.1",
+    "@web/dev-server": "^0.2.1",
+    "@web/test-runner": "^0.16.1",
+    "autoprefixer": "^10.4.14",
+    "babel-loader": "^9.1.2",
     "compression": "^1.7.4",
     "concat": "^1.0.3",
-    "concurrently": "^7.2.2",
+    "concurrently": "^8.0.1",
     "copyfiles": "^2.4.1",
-    "core-js": "^3.23.3",
-    "eslint": "^8.18.0",
-    "eslint-plugin-jsdoc": "^39.3.3",
-    "husky": "^7.0.0",
+    "core-js": "^3.30.1",
+    "eslint": "^8.39.0",
+    "eslint-plugin-jsdoc": "^43.1.1",
+    "husky": "^8.0.3",
     "lodash": "^4.17.21",
-    "markdown-magic": "^2.6.0",
-    "marked": "^4.0.17",
-    "nodemon": "^2.0.18",
+    "markdown-magic": "^2.6.1",
+    "marked": "^4.3.0",
+    "nodemon": "^2.0.22",
     "npm-run-all": "^4.1.5",
-    "postcss": "^8.4.14",
-    "postcss-discard-comments": "^5.1.2",
+    "postcss": "^8.4.23",
+    "postcss-discard-comments": "^6.0.0",
     "postcss-remove-rules": "^1.0.0",
     "postcss-selector-replace": "^1.0.2",
-    "prismjs": "^1.28.0",
-    "rollup": "^2.75.7",
-    "rollup-plugin-minify-html-literals": "^1.2.6",
-    "rollup-plugin-serve": "^2.0.0",
-    "rollup-plugin-terser": "^7.0.2",
-    "sass": "^1.53.0",
-    "semantic-release": "^19.0.3",
-    "sinon": "^14.0.0",
-    "stylelint": "^14.9.1",
-    "stylelint-config-idiomatic-order": "^8.1.0",
-    "stylelint-config-standard": "^26.0.0",
-    "stylelint-config-standard-scss": "^4.0.0",
-    "stylelint-order": "^5.0.0",
-    "stylelint-scss": "^4.2.0",
+    "prismjs": "^1.29.0",
```

Comment:
Remove

Reply:
wait, but then we need to revert changes in [here](https://github.com/AlaskaAirlines/auro-flight/pull/64/commits/037ffee53935530dcf6865a8e3fc649a80d13515), no?

Reply:
Righty, my bad.